app/controllers/application_controller.rb

```diff
@@ -16,6 +16,17 @@ class ApplicationController < ActionController::Base
   around_action :set_timezone
   after_action :store_location
 
+  unless Rails.env.production?
+    around_action :n_plus_one_detection
+
+    def n_plus_one_detection
+      Prosopite.scan
+      yield
+    ensure
+      Prosopite.finish
+    end
+  end
+
```

Comment:
Hmm, I don't think this is how I'd do this, even if I wanted it on in both test and dev -- which i'm also not sure about -- what i would do is probably

```
  before_action :check_forced_logout
  around_action :set_timezone
  around_action :n_plus_one_detection
  after_action :store_location
```

and then down after the other methods
```
  def n_plus_one_detection
    return if Rails.env.production?
    Prosopite.scan
    yield
  ensure
    Prosopite.finish
  end
```
which is i think more in keeping with project style

Reply:
Alright; that's an easy fix.